test-case/multiple-pause-resume.sh

```diff
@@ -86,9 +86,10 @@ do
     fi
 done
 
-# get the min value of TPLG:'pipeline count' with Case:'pipeline count'
-[[ ${#pipeline_idx_lst[*]} -gt ${OPT_VAL['c']} ]] && max_count=${OPT_VAL['c']} || max_count=${#pipeline_idx_lst[*]}
-[[ $max_count -eq 1 ]] && dlogw "pipeline count is 1, don't need to run this case" && exit 2
+max_count="${OPT_VAL['c']}"
+if [ "$max_count" -gt "$PIPELINE_COUNT" ] || [ "$max_count" == "1" ]; then
```

Comment:
Why would the user give a parameter bigger than `$PIPELINE_COUNT`, is this a valid use case? If yes it should be explained in the help and/or comments. If not then it should fail.

Reply:
What happens now when `${#pipeline_idx_lst[*]}` is 1, does the test run? It didn't before this PR.